runtime/jcl/common/jclexception.c

```diff
@@ -126,11 +127,12 @@ getStackTraceIterator(J9VMThread * vmThread, void * voidUserData, J9ROMClass * r
 				J9Module *module = NULL;
 				U_8 *classNameUTF = NULL;
 				UDATA length = 0;
+				/* TODO: this can be null */
```

Comment:
Not sure what this means, but it will prevent me merging...

Reply:
This is flagging a pre-existing issue with this code.  Something I intend to fix in a separate PR.

Reply:
OK